src/scripts/main.js

```diff
@@ -2,7 +2,19 @@
 
 const people = require('./lib/people');
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+const table = document.querySelector('.dashboard');
 
-// write your code here
+[...people].forEach(person => {
+  const row = document.createElement('tr');
+
+  row.innerHTML = `
+    <td>${person.name}</td>
+    <td>${person.sex === 'f' ? 'Female' : 'Male'}</td>
+    <td>${person.born}</td>
+    <td>${person.died}</td>
+    <td>` + (person.died - person.born) + `</td>
+    <td>` + (Math.round(person.born / 100)) + `</td>
```

Comment:
ой прямо дуже дивне і неочевидне форматування, я вже подумав що я щось пропустив, поки зрозумів як це працює)

```suggestion
    <td>${person.died - person.born}</td>
    <td>${Math.round(person.born / 100))}</td>
```